eng/Version.Details.xml

```diff
@@ -5,13 +5,13 @@
       <Uri>https://github.com/dotnet/roslyn</Uri>
       <Sha>d7a4cad21c39e18f3d5e1f7fa7dd3f93668066b4</Sha>
     </Dependency>
-    <Dependency Name="System.CommandLine" Version="2.0.0-beta4.22564.1">
+    <Dependency Name="System.CommandLine" Version="2.0.0-beta4.23219.2">
```

Comment:
We need to bump the version to be in sync with https://github.com/dotnet/sdk/pull/33157

```suggestion
    <Dependency Name="System.CommandLine" Version="2.0.0-beta4.23307.1">
```